lib/repositories/room_repository.dart

```diff
@@ -41,7 +41,7 @@ class RoomRepository {
         return RoomDocument(
           id: snapshot.id,
           // TODO(noboru-i): Check it later about "!".
-          room: Room.fromJson(snapshot.data()!),
+          room: Room.fromJson(snapshot.data() as Map<String, dynamic>),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen_vs_flutter/blob/92c583ad9af22117098fa11a52f80062cc192d63/lib/repositories/room_repository.dart#L44"></span>Don't cast a nullable value to a non nullable type</td>
    </tr>
  </tbody>
</table>